src/mixins/editorMixin.js

```diff
@@ -42,6 +42,15 @@ export default {
 			// Either `pageContent` is filled from editor or we finished fetching it from DAV
 			return !!this.pageContent || !this.loading('pageContent')
 		},
+
+		/**
+		 * Use `this.page` if available (e.g. in `PagePrint`) and fallback to `currentPage`
+		 */
+		pageToUse() {
+			return this.page
+				? this.page
+				: this.currentPage
```

Comment:
might be optimized at build time anyway, but could it not just be like this?
```
this.page || this.currentPage
```

Reply:
Good catch :fishing_pole_and_fish: Fixed now.